editor/components/media-placeholder/index.js

```diff
@@ -86,21 +88,51 @@ class MediaPlaceholder extends Component {
 			onHTMLDrop = noop,
 			multiple = false,
 			notices,
+			hasUploadPermissions,
 		} = this.props;
+		const defaultInstruction = onSelectUrl ? '' : __( 'To edit this block, you need permission to upload media.' );
```

Comment:
I've noted this elsewhere in people's usage, just an FYI that if you don't want anything output, you're best to use `null`, not the empty string, as otherwise React will unnecessarily create an empty `document.createTextFragment`.

See: https://codepen.io/aduth/pen/mKeyzQ

In this case, it doesn't occur because `Placeholder` uses a truthiness check on `instructions` to determine whether to output, but a good rule of thumb all the same.